ee/ucp/authorization/isolate-nodes.md

```diff
@@ -226,20 +226,42 @@ Select the **ops-nodes** namespace, and create a `Full Control` grant for the
 
 ### Link the namespace to a node collection
 
-The last step is to link the Kubernetes namespace the `/Prod` collection.
-
-1.  Navigate to the **Namespaces** page, and find the **ops-nodes** namespace
-    in the list.
-2.  Click the **More options** icon and select **Link nodes in collection**.
+The last step is to link the Kubernetes namespace to the `/Prod` collection. Namespaces are associated with a node collection using the namespace definition information provided in a configuration and an annotation file:
+    
+#### Configuration file
+The configuration file specifies backend behavior in json or yaml using the following format:
+    ```
+    podNodeSelectorPluginConfig:
+     clusterDefaultNodeSelector: name-of-node-selector
+     namespace1: ops-nodes
+    ```
+The `PodNodeSelector` configuration file is specified in the file provided to the API server with the command line flag '--admission-control-config-file` using the following format:
+    ```
+    kind: AdmissionConfiguration
+    apiVersion: apiserver.k8s.io/v1alpha1
+    plugins:
+    - name: PodNodeSelector
+      path: podnodeselector.yaml
+    ```
 
-    ![](../images/isolate-nodes-7.png){: .with-border}
+#### Annotation file
```

Comment:
I  think we should just use this method and try to explain that as long as we define the `scheduler.alpha.kubernetes.io/node-selector: name-of-node-selector` when we create the namespace, all applications deployed in that namespace will be pinned to the nodes with the node selector specified.

Reply:
@nicolaka Added an additional statement in the doc. Thank you!